src/content/en/updates/2019/08/twas-quickstart.md

```diff
@@ -0,0 +1,221 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A guide to get started building a basic, bare-bones Trusted Web Activity.
+
+{# wf_published_on: 2019-08-13 #}
+{# wf_updated_on: 2019-08-13 #}
+{# wf_tags: trusted-web-activity #}
+{# wf_blink_components: N/A #}
+
+# Trusted Web Activities Quick Start Guide {: .page-title }
+
+Trusted Web Activities can be a bit tricky to set up, especially if all you want to do is display
+your website.
+This guide will take you through creating a basic TWA, covering all the gotchas.
+
+First off, you’ll need:
+
+* [Android Studio Installed](https://developer.android.com/studio/install)
+* An Android phone or emulator connected and set up for development
+([enable USB debugging](https://developer.android.com/studio/debug/dev-options.html#enable)if
+you’re using a physical phone).
+* A browser that supports Trusted Web Activities on your development phone. Chrome 72+ will work,
+support in other browsers is on its way.
+
+## Clone and customize the example repo {: #clone-and-customize-the-example-repo }
+
+The [svgomg-twa](https://github.com/GoogleChromeLabs/svgomg-twa) repo contains an example TWA that
+you can customize to launch your website:
+
+Clone the project (`git clone https://github.com/GoogleChromeLabs/svgomg-twa.git`).
+Import the Project into Android Studio, using File > New > Import Project, and select the folder to
```

Comment:
Don't they need to deploy the web app at some point?

Reply:
I think the document assumes they already have web app deployed somewhere. Maybe list this assumption (and any others) on the beginning?

Reply:
I'll add it to the "You'll need" section.